templates/web/card.html

```diff
@@ -1,39 +0,0 @@
-<!doctype html>
```

Comment:
Did I miss somewhere where the OG stuff is being added appropriate to the URL being fetched?

Reply:
root-page-manager.ts is already doing that . Not in this changeset so thats why you dont see it